.github/workflows/main.yml

```diff
@@ -32,24 +32,24 @@ jobs:
       - name: Install cmake
         uses: lukka/get-cmake@v3.18.3
 
-      - name: Check cache for Embedded Arm Toolchain arm-none-eabi-gcc
-        id:   cache-toolchain
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-toolchain-9-2020-q2
-        with:
-          path: ${{ runner.temp }}/arm-none-eabi
-          key:  ${{ runner.os }}-build-${{ env.cache-name }}
-          restore-keys: ${{ runner.os }}-build-${{ env.cache-name }}
-
-      - name: Install Embedded Arm Toolchain arm-none-eabi-gcc
-        if:   steps.cache-toolchain.outputs.cache-hit != 'true'  # Install toolchain if not found in cache
-        uses: fiam/arm-none-eabi-gcc@v1.0.4
+      #- name: Check cache for Embedded Arm Toolchain arm-none-eabi-gcc
+      #  id: cache-toolchain
+      #  uses: actions/cache@v2
+      #  env:
+      #    cache-name: cache-toolchain-10.3-2021.10
+      #  with:
+      #    path: ${{ runner.temp }}/arm-none-eabi
+      #    key: ${{ runner.os }}-build-${{ env.cache-name }}
+      #    restore-keys: ${{ runner.os }}-build-${{ env.cache-name }}
```

Comment:
Couldn't get it working properly and it does not seem required (the toolchain download is cached), I think it's fine to disable?

Reply:
I'm fine with this being disabled for now, we can re-enable it later for the actions speed up later.